File: Problems/CountingTheNumberOfCharacters_2/CountingTheNumberOfCharacters.cpp
#include <iostream>
#include <string>
#include <cstdlib>

using namespace std;

int main()
{
	int lenght;
	string str;

	while (true)
	{
		cout << "Introduce a string: ";
		getline(cin, str);
		lenght = str.length();

		if (str == "")
		{
			str = "Please introduce a correct string.\n";
			cout << str;
			getchar();
			system("cls");
		}
		else
		{
			str = str + " has " + to_string(lenght) + " characters.\n";
			break;
		}
	}

	cout << str;

	return 0;
}

File: Problems/CountingTheNumberOfCharacters_2/CountingTheNumberOfCharacters.py
import os

while True:
    mystr = input("Enter a string: ")
    if mystr == "":
        print("Enter a valid string")
        input("Press Enter to continue...")
        os.system('cls' if os.name == 'nt' else 'clear')
    else:
        mystr = mystr + " has " + str(len(mystr)) + " characters."
        break
print(mystr)